### PR TITLE
Fix radio issues

### PR DESCRIPTION
**Pull request by alexunique0519**
### Description

Fixed the radio issues and some HTML issues following the w3 validator (nu checker)

### CODE REVIEW

File: components/Forms/Radio.tsx [hunk 1]
```diff
@@ -36,54 +36,47 @@ export const Radio: React.VFC<InputProps> = ({
...
       <fieldset>
-        {/* <QuestionLabel
+        <QuestionLabel
           name={name}
           type="radio"
           label={label}
           requiredText={requiredText}
           helpText={helpText}
-        /> */}
-        <legend
```
Comment (MarcoGoC) [inline components/Forms/Radio.tsx:46]: not sure IF not having a legend would pass a11y

Comment (alexunique0519) [reply components/Forms/Radio.tsx:46]: I tried on the nu checker, it didn't complain

Comment (alexunique0519) [reply components/Forms/Radio.tsx:46]: @MarcoGoC  we already have a legend for each card, I don't think we need legend for each radio

**Comment by alex-solo** [comment]
@alexunique0519 
image

Still seeing the styling that we don't want. Is that something you don't see when testing it?

**Review by alex-solo — APPROVED** [review]
seems to be working. Thanks for pushing this along!

**Pull request merged**